build.csx

```diff
@@ -153,7 +153,7 @@ targets.Add(
 
 targets.Add(
     "pack",
-    DependsOn("build", "outputDirectory"),
+    DependsOn("build", "pdbgit"),

```

Comment:
Huh. I removed it from here and tried a build, and it all worked - I think nuget is making the directory itself.
If we're brave, we could drop the target.